crates/spl_network_messages/src/lib.rs

```diff
@@ -23,7 +23,7 @@ pub use visual_referee_message::{VisualRefereeDecision, VisualRefereeMessage};
 pub struct HulkMessage {
     pub player_number: PlayerNumber,
     pub fallen: bool,
-    pub robot_to_field: Isometry2<f32>,
+    pub ground_to_field: Isometry2<f32>,
     pub ball_position: Option<BallPosition>,
```

Comment:
These should be framed...

Reply:
You mean something like this?

```suggestion
    pub pose: Pose<field>,
    pub ball_position: Option<BallPosition>,
```

I thought about that, but this is incompatible with our current crate layout.
We currently have the coordinate systems defined in `types`, which depends on the `spl_network_messages` crate, so we cannot use `Field` here.

Reply:
yes, then we may move the coordinate systems to a separate crate ;)

Reply:
... which opens a new naming problem. 😅 
We already have `crates/coordinate_systems`...

Reply:
Done